src/apps/Pages/Partners/SponsorsList.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import SponsorItem from './SponsorItem';
+
+export default React.createClass({
+
+  displayName: 'Sponsors List',
+
+  propTypes: {
+    sponsors: React.PropTypes.any.isRequired
+  },
+
+  getStyles() {
+    return {
+      componentStyle: {
+        padding: 10,
+        paddingLeft: 20
+      }
+    };
+  },
+
+  render() {
+    let sponsors = [];
+    const styles = this.getStyles();
+
+    this.props.sponsors.forEach(function(sponsor) {
```

Comment:
use `map` instead of `forEach` and put it into inside of return, if you have a questions how to do it just PM me on slack